packages/tea-map-cache/src/tea.ts

```diff
@@ -12,8 +15,16 @@ const log = new Logger({ sender: '@cumulus/tea-map-cache/tea' });
  * @param {number} params.retries     - retries override for pRetry
```

Comment:
If it's optional it should be `[params.retries]`

Reply:
Good catch - 54d702e31

Reply:
54d702e31